client/src/components/Filters/PriceFilter.js

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import ToggleButton from "@mui/material/ToggleButton";
+import ToggleButtonGroup from "@mui/material/ToggleButtonGroup";
+
+const DEFAULT_FILTER = {
+  $: false,
+  $$: false,
+  $$$: false,
+  $$$$: false,
+};
+
+function PriceCategoryFilter(restaurant) {
+  return selectedPrices.includes(restaurant.price_category);
```

Comment:
Variable out of scope. This doesn't compile.

Reply:
remove this for now. The ticket intends for you to add the filter UI only, not the functionality. If you can, thats great. If not, then just leave it out.